inifix/format.py

```diff
@@ -2,44 +2,100 @@
 import os
 import re
 import sys
+import warnings
 from typing import Optional
 from typing import Sequence
 
 from inifix.io import load
 
 NAME_COL_MINSIZE = 10
 COMMENT_COL = 80
+DEFAULT_PADDING_SIZE = 4
+
+
+class Parameter:
+    def __init__(self, name: str):
+        self.name = name
+        self.length = len(name)
+
+    def __str__(self) -> str:
+        return self.name
+
+    def __repr__(self) -> str:
+        return "'" + self.name + "'"
```

Comment:
I'm not convinced this class is necessary yet, but the first three method look a lot like you just want to inherit from `str`
```suggestion
class Parameter(str):
```